### PR TITLE
Smtp emailer issue 5951

### PR DESCRIPTION
**Pull request by Dravere**
First three points of issue #5951 are addressed in this pull-request.

* Fixed Typo at emailer.js line 66 serice => service
* Allow for use without authentication
* Add option for selecting security setting (default should be backward compatible -> set to be encrypted)

The other point will be addressed in a separate pull-request.

### CODE REVIEW

File: src/emailer.js [hunk 1]
```diff
@@ -56,17 +56,37 @@ Emailer.registerApp = function (expressApp) {
...
+			};
+		}
 
-		if (meta.config['email:smtpTransport:serice'] === 'nodebb-custom-smtp') {
+		if (meta.config['email:smtpTransport:service'] === 'nodebb-custom-smtp') {
 			smtpOptions.port = meta.config['email:smtpTransport:port'];
 			smtpOptions.host = meta.config['email:smtpTransport:host'];
-			smtpOptions.secure = true;
+
+			switch(meta.config['email:smtpTransport:security']) {
```
Comment (pitaj) [inline src/emailer.js]: I'd prefer to just use an if-else if-else here, unless you see room for other options in the future.

Comment (Dravere) [reply src/emailer.js]: Changed it to if-else and pushed.

File: src/emailer.js [hunk 2]
```diff
@@ -56,17 +56,35 @@ Emailer.registerApp = function (expressApp) {
...
 			smtpOptions.port = meta.config['email:smtpTransport:port'];
 			smtpOptions.host = meta.config['email:smtpTransport:host'];
-			smtpOptions.secure = true;
+
+			if(meta.config['email:smtpTransport:security'] === 'NONE') {
+				smtpOptions.secure = false;
+				smtpOptions.requireTLS = false;
+				smtpOptions.ignoreTLS = true;
+			}
+			else if(meta.config['email:smtpTransport:security'] === 'STARTTLS') {
```
Comment (pitaj) [inline src/emailer.js]: Make sure you run `npm test` before committing, it catches these kinds of linting errors quickly.

Comment (Dravere) [reply src/emailer.js]: I thought I had eslint activated in the IDE, but obviously it wasn't. `npm test` sadly doesn't produce anything meaningful on my end: `✖ 70937 problems (70933 errors, 4 warnings)` :smile: 

But I activated eslint now and fixed the errors in emailer.js. Sorry about that, pushing again.

**Comment by pitaj** [comment]
LGTM

**Pull request merged**